File: LeetCode/C/1两数之和.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
#include <stdlib.h>//使用malloc
int* twoSum(int* nums, int numsSize, int target) {
    int i;
    int j;
    int *a = (int*)malloc(2*sizeof(int));//此处使用2*sizeof(int) 是让a指向两个int类型的数据
    for(i=0;i<numsSize;i++)
        for(j=i+1;j<numsSize&&j!=i;j++)
            if(nums[i]+nums[j]==target)
            {
                a[0]=i;
                a[1]=j;
            }
    return a;
}

File: LeetCode/Python/1两数之和.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        n = len(nums)
        for x in range(n):
            for y in range(x+1,n):
                if nums[y] + nums[x] == target:
                    return x,y